smarts/core/lidar.py

```diff
@@ -69,7 +69,7 @@ def _compute_static_lidar_noise(self):
                     self._sensor_params.noise_mu, self._sensor_params.noise_sigma
                 )
             )
-        return np.array(static_lidar_noise, dtype=np.float)
+        return np.array(static_lidar_noise, dtype=np.float64)
```

Comment:
I believe `np.float64` is deprecated in the newer versions of `numpy`. You can just use `float` here.

Reply:
It appears that `np.float` is deprecated, and may be replaced with either `float` or the numpy scalar type name `np.float64`. Here, I simply preferred the explicit bit-width sized alias of `np.float64` .

Deprecation notes: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations .